{
	"dataType": "CVE_RECORD",
	"dataVersion": "5.1",
	"cveMetadata": {
		"cveId": "CVE-2020-4958",
		"assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
		"assignerShortName": "ibm",
		"state": "PUBLISHED",
		"datePublished": "2021-01-20T00:00:00",
		"dateUpdated": "2024-08-04T08:14:59.203Z",
		"dateReserved": "2019-12-30T00:00:00"
	},
	"containers": {
		"cna": {
			"providerMetadata": {
				"orgID": "9a959283-ebb5-44b6-b705-dcc2bbced522",
				"shortName": "ibm",
				"dateUpdated": "2021-01-21T14:00:25"
			},
			"descriptions": [
				{
					"lang": "en",
					"value": "IBM Security Identity Governance and Intelligence 5.2.6 does not perform any authentication for functionality that requires a provable user identity or consumes a significant amount of resources. IBM X-Force ID: 192209."
				}
			],
			"affected": [
				{
					"vendor": "IBM",
					"product": "Security Identity Governance and Intelligence",
					"versions": [
						{
							"status": "affected",
							"version": "5.2.6"
						}
					]
				}
			],
			"problemTypes": [
				{
					"descriptions": [
						{
							"type": "text",
							"lang": "en",
							"description": "Gain Access"
						}
					]
				}
			],
			"metrics": [
				{
					"cvssV3_0": {
						"version": "3.0",
						"vectorString": "CVSS:3.0/AV:N/AC:H/I:H/C:H/PR:N/S:U/UI:N/A:N/E:U/RC:C/RL:O",
						"attackVector": "NETWORK",
						"attackComplexity": "HIGH",
						"privilegesRequired": "NONE",
						"userInteraction": "NONE",
						"scope": "UNCHANGED",
						"confidentialityImpact": "HIGH",
						"integrityImpact": "HIGH",
						"availabilityImpact": "NONE",
						"baseScore": 7.4,
						"baseSeverity": "HIGH",
						"exploitCodeMaturity": "UNPROVEN",
						"remediationLevel": "OFFICIAL_FIX",
						"reportConfidence": "CONFIRMED",
						"temporalScore": 6.4,
						"temporalSeverity": "MEDIUM"
					}
				}
			],
			"references": [
				{
					"tags": [
						"x_refsource_CONFIRM"
					],
					"url": "https://www.ibm.com/support/pages/node/6403247"
				},
				{
					"name": "ibm-sva-cve20204958-missing-auth (192209)",
					"tags": [
						"vdb-entry",
						"x_refsource_XF"
					],
					"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/192209"
				}
			],
			"datePublic": "2021-01-20T00:00:00",
			"x_legacyV4Record": {
				"CVE_data_meta": {
					"ASSIGNER": "psirt@us.ibm.com",
					"DATE_PUBLIC": "2021-01-20T00:00:00",
					"ID": "CVE-2020-4958",
					"STATE": "PUBLIC"
				},
				"affects": {
					"vendor": {
						"vendor_data": [
							{
								"product": {
									"product_data": [
										{
											"product_name": "Security Identity Governance and Intelligence",
											"version": {
												"version_data": [
													{
														"version_value": "5.2.6"
													}
												]
											}
										}
									]
								},
								"vendor_name": "IBM"
							}
						]
					}
				},
				"data_format": "MITRE",
				"data_type": "CVE",
				"data_version": "4.0",
				"description": {
					"description_data": [
						{
							"lang": "eng",
							"value": "IBM Security Identity Governance and Intelligence 5.2.6 does not perform any authentication for functionality that requires a provable user identity or consumes a significant amount of resources. IBM X-Force ID: 192209."
						}
					]
				},
				"impact": {
					"cvssv3": {
						"BM": {
							"A": "N",
							"AC": "H",
							"AV": "N",
							"C": "H",
							"I": "H",
							"PR": "N",
							"S": "U",
							"UI": "N"
						},
						"TM": {
							"E": "U",
							"RC": "C",
							"RL": "O"
						}
					}
				},
				"problemtype": {
					"problemtype_data": [
						{
							"description": [
								{
									"lang": "eng",
									"value": "Gain Access"
								}
							]
						}
					]
				},
				"references": {
					"reference_data": [
						{
							"name": "https://www.ibm.com/support/pages/node/6403247",
							"refsource": "CONFIRM",
							"title": "IBM Security Bulletin 6403247 (Security Identity Governance and Intelligence)",
							"url": "https://www.ibm.com/support/pages/node/6403247"
						},
						{
							"name": "ibm-sva-cve20204958-missing-auth (192209)",
							"refsource": "XF",
							"title": "X-Force Vulnerability Report",
							"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/192209"
						}
					]
				}
			}
		},
		"adp": [
			{
				"providerMetadata": {
					"orgID": "af854a3a-2127-422b-91ae-364da2661108",
					"shortName": "CVE",
					"dateUpdated": "2024-08-04T08:14:59.203Z"
				},
				"title": "CVE Program Container",
				"references": [
					{
						"tags": [
							"x_refsource_CONFIRM",
							"x_transferred"
						],
						"url": "https://www.ibm.com/support/pages/node/6403247"
					},
					{
						"name": "ibm-sva-cve20204958-missing-auth (192209)",
						"tags": [
							"vdb-entry",
							"x_refsource_XF",
							"x_transferred"
						],
						"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/192209"
					}
				]
			}
		]
	}
}
